src/sections/Footer/index.tsx

```diff
@@ -75,6 +76,19 @@ const Footer = (): JSX.Element => (
           />
         </a>
       </li>
+      <li>
+        <a
```

Comment:
Just a note, the Mastodon link on nodejs.org is an invisible link meant to the platform to validate that this domain owns that Mastodon account.

What you're doing here differs from what @Trott did on his PR. Which is fine, but I just wanted to point that out.

Reply:
> What you're doing here differs from what @ Trott did on his PR. Which is fine, but I just wanted to point that out.

Yes, I know.